src/components/DataView/components/DataViewHeader/DataViewHeader.tsx

```diff
@@ -1,14 +1,22 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2020 - 2021 Gemeente Amsterdam
+// Copyright (C) 2020 - 2023 Gemeente Amsterdam
+import type { ReactNode } from 'react'
 import { useMemo } from 'react'
 
-import PropTypes from 'prop-types'
-
 import { StyledTHead } from 'components/DataView/styled'
 
 import DataViewHeaderRow from './components/DataViewHeaderRow'
 
-const DataViewHeader = ({ numberOfColumns, headers, filters }) => {
+export interface Props {
```

Comment:
no need to export this type, it's not used anywhere else.

Reply:
Removed it